src/builder/yara_rule_builder.cpp

```diff
@@ -176,8 +176,8 @@ YaraRuleBuilder& YaraRuleBuilder::withComment(const std::string& comment, bool m
  */
 YaraRuleBuilder& YaraRuleBuilder::withStringMeta(const std::string& key, const std::string& value)
 {
-	if (key == std::string{} || value == std::string{})
-		throw RuleBuilderError("Error: String-Meta key and value must be non-empty.");
+	if (key.empty())
```

Comment:
There is going to be a conflict when merging this PR and #59. I think it would be better to wait until #59 is in `master` and then only change `key == std::string{}` to `key.empty()` instead of replicating the work done in #59 (apart from a missing test for the bug fix).

Reply:
I have just resolved the conflict, it wasn't a tough one, but next time I can wait until the conflicting branch is merged. Thank you for your suggestion.